addons/kafka-cluster/templates/cluster.yaml

```diff
@@ -46,6 +48,7 @@ spec:
     {{- else }}
     - name: broker
       componentDefRef: kafka-broker
```

Comment:
componentDefRef can be removed

Reply:
considering the compatibility issues with the ClusterDefinition API in some special cases. this logic will be temporarily retained in one or two versions and will be removed in subsequent iterations.